source/developers/how-tos/add-a-catalog-file.rst

```diff
@@ -0,0 +1,42 @@
+How To Add a Catalog File
+#########################
+
+
+This How-to will walk you through the process of adding a ``catalog-info.yaml``
+file to your repository so that it is properly cataloged in the Open edX instance
+of `Backstage`_.
+
+To learn more about why we use ``catalog-info.yaml`` and backstage, see `OEP-55
+Decision 1`_.
+
+.. _OEP-55 Decision 1: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html
+
+.. _Backstage: https://backstage.openedx.org
+
+Assumptions
+***********
+
+This How-to assumes the following:
+
+* You know how to use Git and GitHub PR Workflows
+
+* You know how to edit YAML files
+
+* You have a local clone of the repository you wish to add a catalog file to.
+
+Steps
+*****
+
+1. Go to your local clone of the repository you want to update.
```

Comment:
These steps seem too basic.  Can't we say, "Add a catalog-info.yaml file to your repo, using the content from sample file..."?